pennylane/wires.py

```diff
@@ -103,6 +103,22 @@ def __eq__(self, other):
             return self.wire_tuple == other.wire_tuple
         return False
 
```

Comment:
Adding those so we can do `assert Wires(['q1, 2]) > Wires([1, 'a'])`, which is needed when we use ``groupby`` in various places to sort observable eigenvalues...

Reply:
See comment in related tests below